blog/2022-07-04-immigrate-erraform-cloud-resources-to-kubeVela.md

```diff
@@ -0,0 +1,136 @@
+---
+title: How to Immigrate Legacy Terraform Cloud Resources to KubeVela
```

Comment:
```suggestion
title: How to Migrate Existing Terraform Cloud Resources to KubeVela
```

Reply:
you should use existing instead of legacy as we regard legacy as useless thing.

Reply:
It's okay to say legacy like `Migrating Legacy Applications To The Cloud`.